src/query/api/v1/handler/json/write.go

```diff
@@ -90,8 +90,13 @@ func newStorageWriteQuery(req *WriteQuery) (*storage.WriteQuery, error) {
 		return nil, err
 	}
 
+	tags := models.Tags{}
```

Comment:
nit: Could preallocate the map size here, `make(models.Tags, len(req.Tags))`

Reply:
Yeah good call, wasn't overly concerned with the perf here since it's kind of a test endpoint but will make the change :)